build(configuration): `module` property within package.json

nissoh opened the pull request:
I'm trying to create a bundled file so it can be served to the browser.
my final bundle ends up with es2015 code which cannot run in older versions of chrome, ff, IE.
is it being forced because of the module property?

My `tsconfig.json` `module` and `target` fields configured to `commonjs` (module code generation)
and `es5`. I removed the `module` field locally within my `node_modules` folder and the end
footprint is `es5` as i configured through my own `tsconfig.json`.

Commit:
build(configuration): `module` property within package.json

I'm trying to create a bundled file so it can be served to the browser.

My `tsconfig.json` `module` and `target` fields configured to `commonjs` (module code generation)
and `es5`. I removed the `module` field locally within my `node_modules` folder and the end
footprint is `es5` as i configured through my own `tsconfig.json`.

nissoh commented:
@TylorS ping (:

TylorS commented:
Thanks for pinging me @nissoh, this had gotten away from me.

I've realized in some of my newer projects, but hadn't gotten back to making adjustments here. The "proper" fix, is to make an adjustment to the [tsconfig.json's target field](https://github.com/mostjs-community/proxy/blob/master/tsconfig.json#L6) needs to be changed to `es5` instead of es2015, leaving just the module to `es2015`. So the code generated is transpiled but imports/exports are kept.

Commit:
build($compile): target es2015 to es5

undo remove of property field 'module', tsconfig target changed from es2015 to es5

Commit:
Merge branch 'package-config' of https://github.com/nissoh/proxy into package-config

Commit:
build($compile): undo remove of property field 'module'

TylorS commented:
Fixed with v3.3.0
